### DISCUSSION TITLE
How to re trigger sub components in  lazy mode after a change to sub components

### DISCUSSION THREAD

**Discussion by kavin404** [TanStack/table#3814] [Q&A]
Hi all, I have used the bellow code to implement sub-components lazy loading.

https://codesandbox.io/s/github/tannerlinsley/react-table/tree/v7/examples/sub-components-lazy

 I have a scenario where I would be adding/ removing/editing a sub-component. I was wondering how I can re-trigger the subcomponents to which will make an API call to retrieve the sub-components list again. 

I cannot force the user to click on the hand icon to retrieve what has been added newly.

**Comment by tannerlinsley** [comment]
This is more of a question of implementation, so I'm going to move this to a discussion.
